specs/rollup-node.md

```diff
@@ -146,6 +146,45 @@ Batch contents:
 
 [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
 
+The guarantee must be held that all transactions containing batches
+being sent to L1 must be valid. This means that users must not be able
+to submit transactions to L2 that make an L1 transaction including it
+be invalid. The only way to do so would be to send a transaction to L2
+that is so large that it would use more than the L1 block's gas limit
+as part of [intrinsic gas][intrinsic-gas]. As of [London][london],
+a zero byte of calldata costs 4 gas while a non zero byte of calldata
+costs 16 gas and the [block gas limit is 30 million][block-size].
+An entire block's worth of gas could be used by a transaction with
+7.5 megabytes of zero byte calldata. Including a transaction
+that uses more than the entire block gas limit as part of it's
+intrinsic gas would cause a liveness failure of L2.
+
+```python
+block_gas_limit = 30_000_000
+max_tx_size = block_gas_limit / 4 # 7500000.0
+```
```

Comment:
Would related this to the paragraph above in some way. As presented, I thought it was a prescription on the max tx size rather than the calculation backing the previous paragraph's point.

Reply:
I'm going to delete this code snippet, its not particularly useful